specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/examples/liveevent-delete.json

```diff
@@ -6,7 +6,7 @@
     "liveEventName": "myLiveEvent1",
     "api-version": "2018-07-01",
     "parameters": {
-      "Id": "/subscriptions/0a6ec948-5a62-437d-b9df-934dc7c1b722/resourceGroups/mediaresources/providers/Microsoft.Media/mediaservices/slitestmedia10/liveevents/myLiveEvent1",
+      "id": "/subscriptions/0a6ec948-5a62-437d-b9df-934dc7c1b722/resourceGroups/mediaresources/providers/Microsoft.Media/mediaservices/slitestmedia10/liveevents/myLiveEvent1",
       "Name": "myLiveEvent1",
```

Comment:
@nschonni , were these caught by a tool? I was curious why name, type and location weren't flagged. The definition in ..\common.json has lowercase names for the Resource.

Reply:
Just running OAV across the repository with the latest version. There may be more, but the newer version catches stuff that wasn't likely flagged when the original files were PR'd.
I just PR example changes that clean up some of the new failures, as long as I'm pretty sure that it wasn't a spec issue.